lib/bugsnag/breadcrumbs/validator.rb

```diff
@@ -26,21 +26,21 @@ def validate(breadcrumb)
         if valid_meta_data_type?(v)
           true
         else
-          @configuration.warn("Breadcrumb #{breadcrumb.name} meta_data #{k}:#{v} has been dropped for having an invalid data type")
+          @configuration.debug("Breadcrumb #{breadcrumb.name} meta_data #{k} has been dropped for having an invalid data type")
```

Comment:
Should you include `v.class` in the log message? (as that shouldn't be sensitive)